File: app/src/main/java/org/school/demoapp/CustomBindAdapter.java
package org.school.demoapp;

import android.widget.ImageView;

import androidx.databinding.BindingAdapter;

import com.squareup.picasso.Picasso;

public class CustomBindAdapter
{
    @BindingAdapter("loadImage")
    public static void loadImage(ImageView imageView, String imageUrl)
    {
        Picasso.get().load(imageUrl).into(imageView);
    }
}

File: app/src/main/java/org/school/demoapp/CustomBindAdapter.kt
package org.school.demoapp

import android.widget.ImageView
import androidx.databinding.BindingAdapter
import com.squareup.picasso.Picasso

object CustomBindAdapter {
    @JvmStatic
    @BindingAdapter("loadImage")
    fun loadImage(imageView: ImageView?, imageUrl: String?) {
        Picasso.get().load(imageUrl).into(imageView)
    }
}